model/userDao.js

```diff
@@ -0,0 +1,64 @@
+const { dataSource } = require('./dataSource');
+
+const createUser = async (
+  email,
+  password,
+  name,
+  phoneNumber,
+  address,
+  gender,
+  birthDate,
+  points
+) => {
+  try {
+    return await dataSource.query(
+      `
+        INSERT INTO users (
+            email,
+            password,
+            name,
+            phone_number,
+            address,
+            gender,
+            birth_date,
+            points
+        ) VALUES (
+          ?, ?, ?, ?, ?, ?, ?, ?
+        )
+      `,
+      [email, password, name, phoneNumber, address, gender, birthDate, points]
+    );
+  } catch (err) {
+    console.log(err);
+    err = new Error(err.message);
```

Comment:
1. err 변수는 catch 때 사용하기 때문에 다른 변수를 사용하는 것이 좋습니다.
2. 데이터베이스와 통신 할 때 발생한 에러 메시지를 그대로 노출 시키는 것은 좋지 않습니다. 적절한 에러 메시지를 개발자가 직접 정의해 주는 것이 좋습니다.
```suggestion
    error = new Error(적절한 메시지);
```